tests/AppBundle/Controller/Admin/BlogControllerTest.php

```diff
@@ -80,21 +80,42 @@ public function testAdminBackendHomePage()
     }
 
     /**
-     * This test changes the database contents by deleting a blog post. However,
+     * This test changes the database contents by creating a new blog post. However,
      * thanks to the DAMADoctrineTestBundle and its PHPUnit listener, all changes
      * to the database are rolled back when this test completes. This means that
      * all the application tests begin with the same database contents.
      */
-    public function testAdminDeletePost()
+    public function testAdminNewPost()
     {
+        $postTitle = 'Blog Post Title '.mt_rand();
+        $postSummary = PostFixtures::getRandomPostSummary();
+        $postContent = PostFixtures::getPostContent();
+
         $client = $this->getAdminClient();
-        $crawler = $client->request('GET', '/en/admin/post/1');
-        $client->submit($crawler->filter('#delete-form')->form());
+        $crawler = $client->request('GET', '/en/admin/post/new');
+        $form = $crawler->selectButton('Create post')->form([
+            'post[title]' => $postTitle,
+            'post[summary]' => $postSummary,
+            'post[content]' => $postContent,
+        ]);
+        $client->submit($form);
 
-        $this->assertSame(Response::HTTP_FOUND, $client->getResponse()->getStatusCode());
+        $this->assertTrue($client->getResponse()->isRedirect());
 
-        $post = $client->getContainer()->get('doctrine')->getRepository(Post::class)->find(1);
-        $this->assertNull($post);
+        /** @var Post $post */
+        $post = $client->getContainer()->get('doctrine')->getRepository(Post::class)->find(31);
```

Comment:
The `31` id should never fail ... but maybe a `findByTitle($postTitle)` would make the code more robust?